service/src/main/java/bio/terra/axonserver/app/configuration/SamConfiguration.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.axonserver.app.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "axonserver.sam")
+public record SamConfiguration(String basePath, String resourceId) {}
```

Comment:
rm. 

i don't think we need Sam in axon (not mentioned in the design doc) and at least not needed for this PR.

Reply:
I think we will need sam to retrieve the pet SA access token for making direct cloud class. These calls fall under the "Other needs" section in the proposal. The api design isn't fleshed out yet so I'm happy to remove sam and re-add when it is needed.

Reply:
but we don't need it for /status and /version